editor.html

```diff
@@ -213,23 +213,39 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                         <path style="fill:currentColor" d="m 387.2,237.3 c -6.6,0 -12.8,1.3 -18.7,3.4 l 0,-97.2 -97.2,0 c 2.1,-5.9 3.4,-12.1 3.4,-18.8 0,-31.1 -25.2,-56.2 -56.2,-56.2 -31,0 -56.2,25.2 -56.2,56.2 0,6.6 1.4,12.9 3.4,18.8 l -97.1,0 0,300 97.2,0 c -2.1,-5.9 -3.4,-12.1 -3.4,-18.8 0,-31.1 25.2,-56.2 56.2,-56.2 31,0 56.2,25.2 56.2,56.2 0,6.6 -1.3,12.9 -3.4,18.8 l 97.2,0 0,-97.2 c 5.9,2.1 12.1,3.4 18.7,3.4 31.1,0 56.2,-25.2 56.2,-56.2 -0.1,-31.1 -25.2,-56.2 -56.3,-56.2 z"/>
                       </svg>
                     </div>
-                    <div class="roundlabel">Blockly</div>
+                    <div class="roundlabel"></div>
                 </a>
                 <a href="#" class="roundbutton action hidden" id="command-snippet"
                     tabindex="6"
                     title="Click to select a snippet (code shortcut)">
                     <div class="roundsymbol">
                         <i class="fa fa-cogs"></i>
                     </div>
-                    <div class="roundlabel">Snippets</div>
+                    <div class="roundlabel"></div>
                 </a>
                 <a href="#" class="roundbutton action hidden" id="command-share"
                     tabindex="7"
                     title="Create a link to share your script">
+                <a href="#" class="roundbutton" id="command-language" tabindex="10" title="Select a language">
```

Comment:
From this line up to line 241 should probably got after the share button. I think right now is inserted *inside* the share button.